packages/web3/README.md

```diff
@@ -81,7 +81,7 @@ If you want to use this component with the connected server but without adding a
 
 The ConnectButton component automatically adds to the local storage info about the connector user used and will automatically call enableWeb3() after rereshing the page. So if user was connected once it will automatically initialize web3 connection(No need anymore to add UseEffect hook for enableWeb3() after refrshing the page)
 
```

Comment:
Could you fix the spelling of `refreshing`... it's incorrect in two places in this line.

Reply:
done